src/services/TestNumberService.ts

```diff
@@ -17,23 +17,29 @@ export class TestNumberService {
 
     /**
      * Creates a new test number in the database.
-     * @param activity - the payload containing the activity
+     * @param maxAttempt - Maximum number of attempts for generating a Test Number
```

Comment:
not really a max - really "the number of attempts" aka "attempt". Max attempts seems to be 5, from line 23